### DISCUSSION TITLE
@ error/convert.c/ConvertImageCommand/3229

### DISCUSSION THREAD

**Discussion by Newbieyy1945** [ImageMagick/ImageMagick#6647] [Help]
### ImageMagick version

ImageMagick 6.9.11-60 Q16 x86_64 

### Operating system, version and so on

Ubuntu 22.041

### Description

I recently installed ImageMagick and ran into a problem While executing the cmd 'convert -resize 50% 20230320_1608256.jpg' 
I received the error-message  '@ error/convert.c/ConvertImageCommand/3229'  . Following advice from
https://stackoverflow.com/questions/64791630/imagemagick-no-images-defined-for-output-filename/69903175#69903175 
I changed my policy.xml to 'no policy at all'  I still have the same error-message. Where can I find more information about this error-code. 
Thanks
Marius

**Comment by snibgo** [comment]
What is the complete error message?

The `convert` command needs both an input and an output. (These can be the same name, to overwrite the file.) You provide only one of these.

**Reply by Newbieyy1945** [reply]
thanks snibgo. After installing ImageMagick I found  'convert -resize 50% 20230320_1608256.jpg' on the internet and understood that the input-file would be overwritten. When I specified  both input and output the result was fine.
